lib/dependencies/HarmonyImportSpecifierDependency.js

```diff
@@ -51,12 +52,14 @@ class HarmonyImportSpecifierDependency extends HarmonyImportDependency {
 		name,
 		range,
 		exportPresenceMode,
+		idRangeStarts,
```

Comment:
It is a breaking change, we can use `idRangeStarts` only as a last arguments, because plugins can use this dependency

Reply:
That would force it to be optional, and the code doesn't work if it's not present.

Reply:
> That would force it to be optional, and the code doesn't work if it's not present.

Yes, we need it, because we can't change it right now due plugins, we can change it for webpack v6, that is why I ask do nothing if they are not present

Reply:
Fixed.  Is there something more to do beyond adding the "TODO webpack 6" comment?